devel/libical/files/patch-nodocs.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 23898b6..fb7e452 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -347,7 +347,7 @@ endif()
+ ################# build subdirs ########################
+ 
+ add_subdirectory(design-data)
+-add_subdirectory(doc)
++# add_subdirectory(doc)
```

Comment:
Well, as this isn't a KDE project I could imagine that upstream probably won't want ECM as an additional dependency.

Anyway, what would one have to add to the `cmake` options? Would it be `-DBUILD_doc=FALSE`